pkg/runtime/config.go

```diff
@@ -95,7 +95,7 @@ type Config struct {
 	DaprBlockShutdownDuration    *time.Duration
 	ActorsService                string
 	RemindersService             string
-	SchedulerAddress             *string
```

Comment:
These changes are for consistency with the other flags. If we want to change this, we should have a separate PR making it all consistent.

Reply:
It is more correct for checking omitted state with nil pointer in Go, please leave as is. Doing all at once will not be reviewed and merged, so best to do over time.